### ISSUE TITLE
API errors

### ISSUE THREAD

**Issue by aniceberg** [elahd/ha-nyc311#4]
Keeps throwing the below error:

```
2022-05-01 10:21:31 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration nyc311 which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-05-01 10:22:23 WARNING (MainThread) [homeassistant.config_entries] Config entry 'NYC 311 Public Services Calendar' for nyc311 integration not ready yet; Retrying in background
2022-05-01 10:27:10 ERROR (MainThread) [custom_components.nyc311] Unexpected error fetching nyc311 data:
File "/usr/local/lib/python3.9/site-packages/nyc311calendar/api.py", line 202, in __async_calendar_update
File "/config/custom_components/nyc311/__init__.py", line 39, in async_update_data
File "/usr/local/lib/python3.9/site-packages/nyc311calendar/api.py", line 161, in get_calendar
File "/usr/local/lib/python3.9/site-packages/nyc311calendar/api.py", line 208, in __async_calendar_update
nyc311calendar.api.NYC311API.UnexpectedEntry
```
Does this have to do with exceeding API limits?

**Comment by aniceberg** [comment]
Latest logs, points to a "STAFF ONLY" error:

This error originated from a custom integration.

```
Logger: custom_components.nyc311
Source: custom_components/nyc311/__init__.py:39
Integration: NYC 311 Public Services Calendar (documentation, issues)
First occurred: 9:34:55 AM (22 occurrences)
Last logged: 10:06:16 AM

Unexpected error fetching nyc311 data:
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/nyc311calendar/api.py", line 202, in __async_calendar_update
    status_id = self.KNOWN_STATUSES[item["status"]]["id"]
KeyError: 'STAFF ONLY'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 190, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 150, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/nyc311/__init__.py", line 39, in async_update_data
    return await api.get_calendar(
  File "/usr/local/lib/python3.9/site-packages/nyc311calendar/api.py", line 161, in get_calendar
    api_resp = await self.__async_calendar_update(start_date, end_date, scrub)
  File "/usr/local/lib/python3.9/site-packages/nyc311calendar/api.py", line 208, in __async_calendar_update
    raise self.UnexpectedEntry from error
nyc311calendar.api.NYC311API.UnexpectedEntry
```

**Comment by elahd** [comment]
Thanks for the report! This is fixed in v0.1, but you'll need to update by finding the integration in your HACS tab, hitting "Update information", then "Redownload":

![image](https://user-images.githubusercontent.com/466460/166290931-c558b23b-c9d1-4285-a2de-102d64b559d7.png)

I changed version formats in a way that doesn't trigger HACS' update notification.
